pkgs/development/python-modules/MeshLabXML/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  lib,
+}:
+
+buildPythonPackage rec {
+  pname = "MeshLabXML";
+  version = "2018.3";
+  name = "${pname}-${version}";
+
+  disabled = (pythonOlder "2.7");
+
+  propagatedBuildInputs = [ ];
+
+  doCheck = false; # This library does not have any tests at this point.
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

Reply:
Ah cool, I didn't know `pythonImportsCheck`. Done.